community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -182,7 +182,8 @@
             setting( "store.interval.log.rotation", DURATION, "10m" );
 
     @Description( "Minimum time (in seconds) after last rotation of the internal log before it may be rotated again." )
-    public static final Setting<Integer> store_internal_log_rotation_delay = setting("store.internal_log.rotation_threshold", INTEGER, "300", min(0), max( Integer.MAX_VALUE ) );
+    public static final Setting<Integer> store_internal_log_rotation_delay =
+            setting("store.internal_log.rotation_delay", INTEGER, "300", min(0), max( Integer.MAX_VALUE ) );
```

Comment:
I see no reason why this wouldn't be a proper duration. It's also backwards compatible since the default unit is seconds. So please update the description as well.